lib/routemaster/drain.rb

```diff
@@ -1,5 +1,5 @@
 module Routemaster
   module Drain
-    VERSION = '2.0.0'
+    VERSION = '2.1.0'
```

Comment:
Usually it's considered bad form to bump version in a branch, but as long as there are no other outstanding PRs I won't insist :)